app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/medication/MedicationDetailsViewModel.kt

```diff
@@ -118,18 +120,32 @@ class MedicationDetailsViewModel @Inject constructor(
                 medicationWithSummaryAndPharmacyDto.dispensingPharmacy.faxNumber
             )
         )
-
+        medicationDetails.add(
+            MedicationDetail(
+                "Direction for use",
```

Comment:
If we are going to display "Direction for use" in UI then in case of multi-language support it won't work.